source/jsonnet/england-wales/ccs_household.jsonnet

```diff
@@ -1,3 +1,19 @@
+local placeholders = import '../../lib/placeholders.libsonnet';
+
+local summaryTitle = {
+  text: 'Visitors staying overnight on {census_date}',
+  placeholders: [
+    placeholders.censusDate,
+  ],
+};
+
+local summaryTitlePersonName = {
+  text: '{person_name}',
+  placeholders: [
+    placeholders.personName,
+  ],
+};
```

Comment:
I think these should be inlined where they're used rather than creating variables. Elevating them here implies they're global as this is the top level of the questionnaire definition. This applies to all other places this has been done.